packages/node_modules/@ciscospark/widget-recents/README.md

```diff
@@ -0,0 +1,274 @@
+
+``` js
+import RecentsWidget from '@ciscospark/widget-recents';
+
+ReactDOM.render(
+  <RecentsWidget
+    accessToken="YOUR_ACCESS_TOKEN"
+  />,
+  document.getElementById('ELEMENT')
+);
+```
+
+### Events
+
+The Recents widget exposes a few events for hooking into widget functionality.
+You can directly add DOM event listener like this:
+
+``` html
+<div
+  class="ciscospark-widget"
+  data-toggle="ciscospark-recents"
+  data-access-token="YOUR_ACCESS_TOKEN"
+  />
+<script>
+  document.getElementById('ciscospark-widget').addEventListener('EVENT_NAME', function(event) {
+    // Handle the event here
+    console.log(event.detail);
+  });
+</script>
+```
+
+If you are using *browser globals*, you can provide a callback parameter that will fire whenever any event occurs. You can filter the actions using the name provided like this:
+
+``` js
+var widgetEl = document.getElementById('my-ciscospark-widget');
+// Init a new widget
+ciscospark.widget(widgetEl).recentsWidget({
+  accessToken: 'YOUR_ACCESS_TOKEN',
+});
+
+function callback(name, detail) {
+  if (name === 'messages:created') {
+    // Perform an action if a new message has been created
+  }
+}
+```
+
+Or you can use the (`ampersand-events`)[https://github.com/AmpersandJS/ampersand-events] API to listen to events like this:
+
+``` js
+var widgetEl = document.getElementById('ciscospark-widget');
+ciscospark.widget(widgetEl).on('messages:created', function(e) {
+  console.log(e.detail);
+});
+```
+
+Currently available events:
+
+* `messages:created` - A new message has been posted
+* `messages:unread` - The conversation has unread messages
+* `messages:read` - The conversation's messages have been read
+* `calls:ringing` - An incoming call is ringing
+* `calls:connected` - A call has been answered is now connected
+* `calls:disconnected` - A active call has been disconnected
+
+When handling one of the events listed above we will provide a `event.detail` object that looks something like this:
+
+``` js
+{
+  resource: 'messages',
+  event: 'created',
+  actorId: 'Y2lzY29zcGFyazovL3VzL1BFT1BMRS8xZjdkZTVjYi04NTYxLTQ2NzEtYmMwMy1iYzk3NDMxNDQ0MmQ',
+  data: {
+    id: 'Y2lzY29zcGFyazovL3VzL01FU1NBR0UvMzIzZWUyZjAtOWFhZC0xMWU1LTg1YmYtMWRhZjhkNDJlZjlj',
+    spaceID: 'Y2lzY29zcGFyazovL3VzL1JPT00vUk9PTUlEMDAtUk9PTUlEMDAtUk9PTUlEMDAtUk9PTUlEMDA',
+    object: {
+      displayName: 'Message 123',
+    },
+    published: '2016-12-04T17:33:56.767Z'
+  }
+};
+```
+
+* `resource` - the resource the data is about.
+* `event` - the type of event that was fired.
+* `actorId` - the personId of the user that caused the event. For example, on a messsage created event, the author of the message will be the actorId.
+* `data` - the resource that triggered the event. For example, when `messages` are `created`, the `data` will contain the payload of a message resource.
+
+
+## Browser Support
+
+This widget supports the follow browsers messaging:
```

Comment:
I like to use "is tested against" instead of "supports", but that wouldn't necessarily be true yet.